lua/astrocommunity/lsp/lsp-inlayhints-nvim/init.lua

```diff
@@ -1,6 +1,7 @@
 return {
   {
     "lvimuser/lsp-inlayhints.nvim",
+    event = "User AstroLspSetup",
```

Comment:
```suggestion
```

This lazy loading is handled by the autocmd and should not be loaded on event. This is less performant